plugins/catalog-import/src/components/StepPrepareCreatePullRequest/StepPrepareCreatePullRequest.tsx

```diff
@@ -82,7 +86,7 @@ function generateEntities(
     },
     spec: {
       ...e.spec,
-      owner,
+      ...(owner ? { owner } : {}),
```

Comment:
Why not useCodeowners ? {} : { owner }?

Reply:
I could, but was just avoiding bringing the that concept into this pure function.